scripts/cluster/yarn_cluster_setup/setup.py

```diff
@@ -0,0 +1,414 @@
+import os
+import sys, socket
+
+def write_hadoop_config_file(name, xml):
+	f = open("/usr/local/hadoop/etc/hadoop/" + name, "w")
+	f.write(xml)
+	f.close()
+
+def write_spark_config_file(name, xml):
+	f = open("/spark-3.0.0-SNAPSHOT-bin-SparkFHE/conf/" + name, "w")
+	f.write(xml)
+	f.close()
+
+master_file = open("master", "r")
+slaves_file = open("slaves", "r")
+master_address = master_file.read().strip()
+slaves_address = slaves_file.read().strip()
+
+master_file.close()
+slaves_file.close()
+
+os.system("apt-get update -y && apt-get install python -y && apt-get install -y default-jdk && apt-get install -y curl && apt-get install -y maven && apt-get install -y python-pip && pip install pyhdfs")
+
+# Clear previous installation
+os.system("rm -rf /usr/local/hadoop-*/ && unlink /usr/local/hadoop && rm -rf /data/hadoop/")
+os.system("sed -i /JAVA_HOME/d /root/.bashrc && sed -i /HADOOP_HOME/d /root/.bashrc && sed -i /hadoop/d /root/.bashrc && sed -i /StrictHostKeyChecking/d /etc/ssh/ssh_config")
+
+# Make Global Variables
+os.system("echo 'export JAVA_HOME=/usr/lib/jvm/default-java/' >> /root/.bashrc")
+os.system("echo 'export HADOOP_HOME=/usr/local/hadoop' >> /root/.bashrc")
+os.system("echo 'export PATH=$PATH:/usr/local/hadoop/bin/:/usr/local/hadoop/sbin/' >> /root/.bashrc")
+
+# Configure SSH
+os.system("echo 'StrictHostKeyChecking no' >> /etc/ssh/ssh_config")
+
+# Configure Data Folder for Hadoop
+os.system("mkdir -p /data/hadoop/node && mkdir -p /data/hadoop/data && mkdir -p /data/hadoop/name")
+
+if not os.path.exists("/hadoop-2.9.2.tar.gz"):
```

Comment:
specific version should be referenced by variable and define at the beginning so that we can change later

Reply:
Yes. This script has undergone some revisions. I would love to hear your thoughts on the latest code.